src/wmnds/patterns/content-cards/inline-navigation/_inline-navigation.njk

```diff
@@ -0,0 +1,29 @@
+{% macro wmndsContentCardInlineNavigation(params) %}
+{# imports #}
+{% from "wmnds/components/content-card/_content-card.njk" import wmndsContentCard %}
+{# Set #}
+{% set title = "Page contents" if params.withCurrentPage else "Related content" %}
+{# Set up nav links #}
+{% set navLinks = ["Information for drivers", "Information for bus passengers", "Information for tram passengers", "Information for pedestrians", "Getting  to venues and attractions"] %}
+
+{% call wmndsContentCard() %}
+    <nav class="wmnds-p-sm">
+        <h3 class="wmnds-content-card__title">{{title}}</h3>
+        <ul class="wmnds-content-card__list wmnds-m-b-none">
+            {#- Current page -#}
+            {% if params.withCurrentPage %}
+            <li class="wmnds-content-card__list-item">
+                <span class="wmnds-inset-text" aria-current="page">Overview</span>
```

Comment:
I would import the inset text as a njk file. So if it ever updates, this pattern is kept in sync.

Reply:
The only reason I haven't is because I wouldn't be able to add the `aria-current="page"` attribute